docs/rewind/channel-edit-pane.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 6
+---
+
+# Channel Edit Pane
+![Channel Edit Pane](/img/rewind/edit-pane.png)
+
+The channel edit pane can be used to configure the channels in your show.
+
+Each channel can optionally be labelled and have a colour assigned. If assigned,
+they will be shown in the [Channel Selection Region](channel-selection-region):
+![Channel Selection Button with label and colour](/img/rewind/ui-channel-button.png).
+
+You can copy and paste channel name data directly into *Rewind* from your favourite
+spreadsheet application. What a truly Excellent feature.
```

Comment:
```suggestion
spreadsheet application.
```

Reply:
I thought this was hilarious :'(

Reply:
It's certainly funny if you know how complicated it is to implement ahahahah

Reply:
aha yes dw 'tis cut